tests/export/test_docx.py

```diff
@@ -24,34 +24,35 @@ def convert(path, *args, **kwargs):
 
 class ConvertDocxToHtmlTestCase(DocXFixtureTestCaseFactory):
     cases = (
-        'read_same_image_multiple_times',
         'all_configured_styles',
+        'export_from_googledocs',
+        'external_image',
+        'has_missing_image',
+        'has_missing_image',
         'has_title',
         'inline_tags',
-        'has_missing_image',
         'justification',
         'list_in_table',
-        'external_image',
-        'export_from_googledocs',
-        'has_missing_image',
         'lists_with_styles',
         'missing_numbering',
         'missing_style',
         'nested_lists',
         'nested_table_rowspan',
         'nested_tables',
         'no_break_hyphen',
+        'read_same_image_multiple_times',
+        'rotate_image',
         'shift_enter',
         'simple',
         'simple_lists',
         'simple_table',
         'special_chars',
         'styled_bolding',
         'table_col_row_span',
+        'table_with_multi_rowspan',
         'tables_in_lists',
+        'textbox',
```

Comment:
I would disagree. Way back when, I wanted to have an actual docx file that was being tested for each case we were working on. It's very easy to build an XML snippet that can't actually exist in a docx file. These are smoke tests.